packages/venia-ui/lib/components/FormError/formError.js

```diff
@@ -26,7 +26,7 @@ const FormError = props => {
 
     return errorMessage ? (
         <div className={classes.root} ref={errorRef}>
-            <span className={classes.errorMessage}>{errorMessage}</span>
+            <div className={classes.errorMessage}>{errorMessage}</div>
```

Comment:
Why was this needed?

Reply:
Good callout. If the content is prose, it should probably be `<p>`. If it's more of a label or snippet, `<span>` was correct.

Reply:
IIRC I switched from `span` to `div` because the left border styling wasn't applying correctly on the root. Since the border no longer applies to the form error (per @schensley's mocks) I can revert back to `span`.

Reply:
Resolved in cbd59e3